examples/nlp/token_classification.py

```diff
@@ -0,0 +1,127 @@
+from datasets import load_dataset
```

Comment:
I think we can go ahead and start filling out the rough text for this guide anytime. So introduce the text classification task, talk about the dataset, why we need to preprocess it for the BERT subword tokenizer in particular.

Even if we end up tweaking the preprocessing routines a bit during review, the overall flow and intro description can probably stay fairly static!

Reply:
Sounds good @mattdangerw .